corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -150,6 +152,48 @@ def test_sort_cache_suite(self, *args):
             "./detail[@id='m0_case_short']"
         )
 
+    def test_case_search_action(self):
+        app = Application.wrap(self.get_json('suite-advanced'))
+        app.modules[0].search_config = CaseSearch(
+            search_label=CaseSearchLabel(
+                label={'en': 'Get them'},
+                media_image={'en': "jr://file/commcare/image/1.png"},
+                media_audio={'en': "jr://file/commcare/image/2.mp3"}
+            ),
+            search_again_label=CaseSearchAgainLabel(
+                label={'en': 'Get them'},
+                media_audio={'en': "jr://file/commcare/image/2.mp3"}
+            ),
+            properties=[CaseSearchProperty(name='name', label={'en': 'Name'})],
+        )
+        # wrap to have assign_references called
+        app = Application.wrap(app.to_json())
+
+        # test for legacy action node for older versions
+        self.assertXmlPartialEqual(
+            self.get_xml('case-search-with-action'),
+            app.create_suite(),
+            "./detail[@id='m0_case_short']/action"
+        )
+        self.assertXmlPartialEqual(
+            self.get_xml('case-search-again-with-action'),
+            app.create_suite(),
+            "./detail[@id='m0_search_short']/action"
+        )
+
+        # test for localized action node for new versions
```

Comment:
"new" versions 😉

Reply:
should that be newer? :package:

Reply:
updated to better documentation and also to trigger a test build :)